src/resources/Extensions/ExtensionsInterfaces.ts

```diff
@@ -3,57 +3,208 @@ import {DataStreamType, ExtensionLanguageType} from '../Enums.js';
 
 type Version = 'v1' | 'v2';
 
+export interface ExtensionHealthModel {
+    /**
+     * A qualitative health indicator.
+     *
+     * **Allowed values:**
+     * - `GOOD`
+     * - `WARNING`
+     * - `PROBLEMATIC`
+     * - `UNKNOWN`
+     */
+    healthIndicator: string;
+    /**
+     * Additional details if healthIndicator value is `WARNING` or `PROBLEMATIC`.
+     */
+    reason: string;
```

Comment:
Should we use string unions instead?
```suggestion
     */
    healthIndicator: 'GOOD' | 'WARNING' | 'PROBLEMATIC' | 'UNKNOWN';
    /**
     * Additional details if healthIndicator value is `WARNING` or `PROBLEMATIC`.
     */
    reason: string;
```